Source/Core/Core/PowerPC/JitArm64/Jit.cpp

```diff
@@ -608,6 +614,8 @@ void JitArm64::Jit(u32)
   JitBlock* b = blocks.AllocateBlock(em_address);
   DoJit(em_address, b, nextPC);
   blocks.FinalizeBlock(*b, jo.enableBlocklink, code_block.m_physical_addresses);
+  Common::JITPageWriteDisableExecuteEnable();
+
```

Comment:
This blank line looks like you wanted to type something else, but didn't. We usually don't have empty lines at the end of a method (only between blocks where it makes sense to have some spacing for logical separation)

Reply:
Fixed